constellation-app/constellation
# Feature/issue#296

### Description of the Change

Removed code preventing update of attributes displayed in attirbute editor in cases such as graphj move/rotate and node move.

### Alternate Designs

nil

### Why Should This Be In Core?

bug fix.

### Benefits

Moving nodes, panning, rotating graph now reflects in the attribute editor.

### Possible Drawbacks

Could not identify why the original code (that has been removed) was added in the first case due to no history. Testing as described in associated issue showed no problems.

### Verification Process

1. Open a graph
2. Open the Attribute Editor an expand the Node block
3. Select a Node on the graph and note its X/Y position in the Attribute Editor
4. Move this node and observe its X/Y position changes after completing move
5. Open the Graph element of the atribute editor
6. Pan the graph and confirm camera attribute changes
7. Rorate the graph and confirm camera attribute changes

### Applicable Issues

https://github.com/constellation-app/constellation/issues/296
